packages/@okta/vuepress-site/docs/guides/build-sso-integration/create-your-app/oidc/create.md

```diff
@@ -0,0 +1,89 @@
+### Create an OIDC integration
+
+If you don't have an Okta developer account, begin by signing up for one at [https://developer.okta.com/signup/](https://developer.okta.com/signup/).
+
+1. After you request the developer account and have received the initialization email, click the link to go to your developer org. Sign in as a user with administrative privileges.
+1. Navigate to the Admin Console in your Okta org by clicking **Admin**.
+  ![Admin Button](/img/oin/scim_end-user-ui.png "Admin Button")
+1. If you are in the Developer Console, click **Developer Console** and then **Classic UI** to switch over to the Admin Console in your Okta org.
+  ![Switch to Admin Console](/img/oin/scim_switch-ui.png "Switch to Admin UI")
+1. Click **Applications** > **Applications**.
```

Comment:
Yes - there's a little bit of jumping through hoops to make sure they're in the right console and on the right screen to create the apps and we don't want to lose them along the way. After we have the updated developer on-boarding process that sends them directly to the right spot, this will be unnecessary, and I'll reduce or eliminate them at that point.